x11-libs/goocanvas/goocanvas-2.0.4-r2.ebuild

```diff
@@ -0,0 +1,59 @@
+# Copyright 1999-2022 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+PYTHON_COMPAT=( python3_{8..10} )
+
+inherit gnome2 python-any-r1
+
+DESCRIPTION="Canvas widget for GTK+ using the cairo 2D library for drawing"
+HOMEPAGE="https://wiki.gnome.org/GooCanvas"
+
+LICENSE="LGPL-2"
+SLOT="3.0"
+KEYWORDS="~alpha ~amd64 ~arm64 ~ia64 ~ppc ~ppc64 ~sparc ~x86"
+IUSE="examples +introspection"
+
+RDEPEND="
+	>=x11-libs/cairo-1.10.0
+	>=dev-libs/glib-2.28.0:2
+	>=x11-libs/gtk+-3.0.0:3
+	introspection? ( >=dev-libs/gobject-introspection-0.6.7:= )
+"
+DEPEND="
+	${RDEPEND}
+	${PYTHON_DEPS}
+	>=sys-devel/gettext-0.19.4
+	dev-util/glib-utils
+	>=dev-util/gtk-doc-am-1.16
+	virtual/pkgconfig
+"
+
+src_prepare() {
+	# Do not build demos
+	sed -e 's/^\(SUBDIRS =.*\)demo\(.*\)$/\1\2/' \
+		-i Makefile.am Makefile.in || die "sed failed"
+
+	# Python bindings are built/installed manually, but not at all anymore (py2).
+	sed -e "/SUBDIRS = python/d" -i bindings/Makefile.am bindings/Makefile.in
```

Comment:
Missing `|| die`.

Reply:
Already done, I've noticed it just before your comment ':)